POLYGLOT_README.md

```diff
@@ -211,23 +211,6 @@ The output JSON is the serialization of- [`PiranhaOutputSummary`](/src/models/pi
 
 *It can be seen that the Python API is basically a wrapper around this command line interface.*
 
```

Comment:
```suggestion

### Languages Supported

| Language           | Structural <br> Find-Replace | Chaining <br> Structural Find-Replace | Stale Feature Flag <br> Cleanup      |
|--------------------|------------------------------|---------------------------------------|--------------------------------------|
| **Java**           | :heavy_check_mark:            | :heavy_check_mark:                    | :heavy_check_mark:                   |
| **Kotlin**         | :heavy_check_mark:            | :heavy_check_mark:                    | :heavy_check_mark:                   |
| **Java + Kotlin**  | :x:                           | :x:                                   | :x:                                  |
| **Swift**          | :heavy_check_mark:            | :construction:                        | :construction:                       |
| **Go**             | :heavy_check_mark:            | :heavy_check_mark:                    | :heavy_check_mark:                   |
| **Python**         | :heavy_check_mark:            | :x:                                   | :x:                                  |
| **TypeScript**     | :heavy_check_mark:            | :x:                                   | :x:                                  |
| **TypeScript+React**| :heavy_check_mark:           | :x:                                   | :x:                                  |

```

Reply:
Lets keep the source of truth for language support as the source code. That avoids things getting out of date.